src/OrbitGl/TimerTrack.h

```diff
@@ -46,6 +46,13 @@ struct DrawData {
   float z;
   bool is_collapsed;
 };
+
+struct Rect {
```

Comment:
Nit: `Rectangle`, as per [style guide?](https://google.github.io/styleguide/cppguide.html#General_Naming_Rules)?

Reply:
`Rect` is widely accepted in graphics / UI frameworks as a shortening of Rectangle and we also use this elsewhere. For consistency and brevity I'd keep `Rect`.

Reply:
I actually removed the struct altogether when further cleaning up ThreadTrack::UpdatePrimitives.